lib/index.js

```diff
@@ -83,10 +83,14 @@ PostHTML.prototype.process = function (tree, options) {
    */
   options = options || {}
 
-  if (options.parser) parser = options.parser
-  if (options.render) render = options.render
+  if (options.parser && typeof options.parser === 'function') {
+    parser = options.parser
+  }
+  if (options.render && typeof options.render === 'function') {
+    render = options.render
+  }
 
-  tree = options.skipParse ? tree : parser(tree)
+  tree = typeof tree === 'object' ? tree : parser(tree)
```

Comment:
oh... yes. Removing skipParse is a realy breaking change. And i not sure that checking is good idea.